.gitignore

```diff
@@ -1,4 +1,5 @@
 application/config/database.php
+application/logs/*.php
 .DS_Store
```

Comment:
Maybe we can cache to gitignore also?

Reply:
If we don't enable it by adding `$this->output->cache($n);` it is not used. Nothing is written in there.
See: https://codeigniter.com/userguide3/general/caching.html
So, I don't think is is necessary. What do you think?

Reply:
Adding it is just a being cautious, that even if that is enabled it would not show up in git changes